src/lib.rs

```diff
@@ -138,3 +188,16 @@ enum Reactor {
     Handle(reactor::Handle),
     Remote(reactor::Remote),
 }
+
+cfg_if! {
+    if #[cfg(feature = "tls")] {
+        extern crate tokio_tls;
+        extern crate native_tls;
+
+        /// Re-exported TLS-related types
```

Comment:
Because we have public interfaces now that use all these types, except `Pkcs12`, but you need that type to make the `TlsAcceptor` itself.